.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: ruby
 rvm:
-  - 2.1.0
+  - 2.4.0
+  - 2.3.3
+  - 2.2.6
+  - 2.1.10
```

Comment:
Ah, okay, makes sense.

Or even better: What do you think about adding the Ruby versions in a separate PR first? Then I'll create the PR.